lab-teddy/test/vehicle-test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Car = require('../model/vehicles');
+const expect = require('chai').expect;
+
+describe('vehicles module', function(){
+  describe('when creating a new vehicle object', function(){
+    this.newCar = new Car('lambo', 'super car');
```

Comment:
This test fails because you aren't passing all the properties. I would look at your if statement in your model